css/index.css

```diff
@@ -46,31 +46,76 @@ table {
 	border-collapse: collapse;
 	border-spacing: 0;
 }
+/*----------------------------------------------------*/
+
+
+
 
 * {
     box-sizing: border-box;
 }
+html{
+    font-size:62.5%;
+
+}
 
 html, body {
     height: 100%;
     font-family: 'Roboto', sans-serif;
 }
 
 h1, h2, h3, h4, h5 {
-    font-size: 18px;
+    font-size: 1.8rem;
     margin-bottom: 15px;
     font-family: 'Rubik', sans-serif;
 }
 
 p {
     line-height: 1.4;
 }
-
+body{
+    background: black;
+}
 .container {
-    width: 800px;
+    padding:2% 7% 7% 7%;
+    width:100%;
+    max-width: 1000px;
+    height: 1000px;
+   
```

Comment:
We went in depth in regard to the width difference and why max-width should be 800px to _contain_ everything.